contribs/gmf/src/controllers/AbstractAppController.js

```diff
@@ -726,4 +699,43 @@ gmf.AbstractAppController.prototype.getLocationIcon = function() {
   return arrowWrapper;
 };
 
-gmf.module.controller('AbstractController', gmf.AbstractAppController);
+
+gmf.controllers.AbstractAppController.module = angular.module('GmfAbstractAppControllerModule', [
+  'gettext',
+  'tmh.dynamicLocale',
+  gmf.authentication.module.name,
+  gmf.backgroundlayerselector.component.name,
+  gmf.datasource.module.name,
+  gmf.disclaimer.component.name,
+  gmf.filters.module.name,
+  gmf.layertree.module.name,
+  gmf.map.module.name,
+  gmf.query.extraModule.name,
+  gmf.search.module.name,
+  gmf.theme.module.name,
+  ngeo.message.displaywindowComponent.name,
+  ngeo.misc.extraModule.name,
+  ngeo.query.MapQuerent.module.name,
+  ngeo.query.mapQueryComponent.name,
+  ngeo.statemanager.module.name,
+]);
+
+
+gmf.controllers.AbstractAppController.module.controller('AbstractController', gmf.controllers.AbstractAppController);
+
+
+gmf.controllers.AbstractAppController.module.value('ngeoExportFeatureFormats', [
+  ngeo.misc.FeatureHelper.FormatType.KML,
+  ngeo.misc.FeatureHelper.FormatType.GPX
```

Comment:
Missing `goog.require('ngeo.misc.FeatureHelper');`

Reply:
Weird, shouldn't this dependency be covered by ngeo.misc.extraModule (which requires ngeo.misc.FeatureHelper)?

Reply:
Not in webpack mode because the import is changed as:
`import ngeoMiscFeatureHelpper from 'ngeo/misc/FeatureHelper.js';`
and this:
`ngeoMiscFeatureHelpper.FormatType.GPX`
This mean that with webpack / es6 modules the window namespace isn't filled.
Then the direct usage should steel imported :-)